src/Cart.java

```diff
@@ -0,0 +1,6 @@
+import java.util.HashMap;
+
+public class Cart {
+  private HashMap<Long,String> purchases;
+  private long totalPrice;
```

Comment:
subtotal price may be
so your total price = subtotal + tax (at the order) + shipment fee (can be assigned as shipment method at the order)

Reply:
where is update?